electron/main.ts

```diff
@@ -0,0 +1,207 @@
+import { BrowserWindow, net, ipcMain } from 'electron'
+import path from 'path'
+import { fileExists, readFile, createFile } from '@sasjs/utils'
+import log from 'electron-log'
+import url from 'url'
+import { exec } from 'child_process'
+
+export default class Main {
+  static mainWindow: Electron.BrowserWindow
+  static application: Electron.App
+  static BrowserWindow
+  static serverApiEnvPath = path.join(__dirname, '..', '..', '..', '..', '.env')
```

Comment:
I suggest that you should have a separate file in the utils folder that should export all these file paths.
This comment is for all the `path.join` statements in this file or any other place int the project.